plugins/org.jboss.tools.jst.js.npm/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/target/
```

Comment:
No need to add bin and target to the .gitignore. They both are ignored globally for the jst repo (and other jbt repos too btw).